wbooks/src/app/screens/booklist/components/BookItem/index.js

```diff
@@ -1,21 +1,30 @@
 import React, { Component } from 'react';
 import { View, Image, Text } from 'react-native';
 import styles from './styles';
+import PropTypes from 'prop-types';
 
 class BookItem extends Component {
 
-	render() {
-		return (
+  render() {
+    const {uri, title, author} = this.props;
```

Comment:
Just add a space between, It should look like this: `const { uri, title, author } = this.props;`

Reply:
done!